misc/librenms-irc.service

```diff
@@ -0,0 +1,12 @@
+# /etc/systemd/system/librenms-irc.service 
+
+librenms-irc.service
+[Unit]
+Description=Librenms IRC bot
+
+[Service]
+ExecStart=/opt/librenms/irc.php
```

Comment:
Your guess is right for rpm based systems. Having a config file for not letting people to fiddle with init files (or unit files whatever you call it) is the right way, however, I think we should leave this to package maintainers, there's no distro neutral location that I know of.